src/Uno.UI/UI/Xaml/FrameworkElement.wasm.cs

```diff
@@ -465,7 +476,8 @@ public double MaxWidth
 				typeof(FrameworkElement),
 				new FrameworkPropertyMetadata(
 					defaultValue: double.PositiveInfinity,
-					options: FrameworkPropertyMetadataOptions.AutoConvert | FrameworkPropertyMetadataOptions.AffectsMeasure
+					options: FrameworkPropertyMetadataOptions.AutoConvert | FrameworkPropertyMetadataOptions.AffectsMeasure,
+					propertyChangedCallback: OnGenericPropertyUpdated
```

Comment:
I'm wondering if we should only be allowing this in DEBUG. Most of the time this will do nothing.

Reply:
Right now yes, but we should probably be calling InvalidateMeasure() here? Per other platforms. (I didn't add it because I didn't want to include that functional change here.)

Reply:
Ah never mind - I see that it's already set `AffectsMeasure`

Reply:
Fixed